docs/index.html

```diff
@@ -21,44 +21,44 @@
 					<img src="https://i.imgur.com/vKFMlIj.png" alt="l5 logo" width="50%" style="text-align: none;" />
 				</section>
 
-				<section>
-					<h2>From Zero to Auto Pilot:</h2>
-					<h2>Exploring Kubernetes Operator Capability Levels</h2>
-				</section>
-
 
 				<section>
-					<h3><b>What is an <span style="color:rgb(212, 18, 18);">operator</span> ?</b></h3>
-					<p class="fragment fade-up" style="color:rgba(255, 255, 255, 0.75);">An <u>opertator</u> is a method of packaging, deploying, and managing an application by extending the functionality of the Kubernetes API </p>
+					<h2>From Zero to <span style="color:red">Auto Pilot</span>:</h2>
+					<p>Exploring Kubernetes Operator Capability Levels</p>
 				</section>
 
+
 				<section data-auto-animate>
-					<h3>What is the L5 operator &</h3>
+					<h3>What is the L5 operator <a href="https://bestie-rescue.herokuapp.com/">&</a></h3>
```

Comment:
I think that's something we should leave for level 5..? I can mention that "the l5 op is an operator with autopilot capabilities that manages a flask . . ." but I think we should maybe level it for Yuri to define, because we each are defining a level of an operator, what their capabilities are, and how we incorporated it into l5 no?

Reply:
we want to define the goal at the beginning i.e. what we are building up to i think

Reply:
added it to the end as transition into operator framework so I can elaborate more how those requirements and capabilities of autopilot that we are aiming towards came about